### DISCUSSION TITLE
Core 2.1

### DISCUSSION THREAD

**Discussion by elanderson** [Dotnet-Boxed/Templates#207] [Q&A]
Looking at the code for the GraphQL template it looks like the project has been upgraded to .NET Core 2.1, but it hasn't been part of a release yet. Do you have any thoughts about when a release including this change might come out?

**Comment by RehanSaeed** [comment]
Hope to get it out today/tomorrow. Thanks for asking, cool blog BTW.

**Comment by RehanSaeed** [comment]
It's building and will be released as I write this.

**Comment by elanderson** [comment]
Thank you! Appreciate all the work you put it to this project.
